app/workflow_executors/step_executors/impl/open_ai_next_js_code_generation_executor.go

```diff
@@ -535,14 +542,6 @@ func (openAiCodeGenerator *OpenAiNextJsCodeGenerator) GetMessages(systemPrompt s
 					Type: "text",
 					Text: fmt.Sprintf("User Feedback: %s", instruction["feedback"]),
 				},
-				{
```

Comment:
why removing it?

Reply:
existing code was going twice in single call to llm, first one with prompt where we replace existing code and second in the messages; i verified this with ayush